meta-networking/recipes-connectivity/sshpass/sshpass_1.09.bb

```diff
@@ -2,9 +2,11 @@ DESCRIPTION = "Non-interactive ssh password auth"
 HOMEPAGE = "http://sshpass.sourceforge.net/"
 SECTION = "console/network"
 LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://COPYING;md5=94d55d512a9ba36caa9b7df079bae19f"
 
 SRC_URI = "${SOURCEFORGE_MIRROR}/sshpass/sshpass-${PV}.tar.gz"
```

Comment:
What is the value of `SOURCEFORGE_MIRROR` here? And who owns it?

Reply:
It's defined in [`meta/conf/bitbake.conf` in openembedded-core](https://github.com/ni/openembedded-core/blob/nilrt/master/hardknott/meta/conf/bitbake.conf#L648) along with other common base URLs.  
Looking at the history, it appears that these MIRROR variables have a couple more uses than the obvious DRY rule:
 - allow people to use geographically nearby download sources
 - [avoid unnecessary cache invalidations](https://github.com/ni/openembedded-core/commit/168f3d71bcfc9daa2984c21f35e2a2fdf754c57c)

Reply:
OK. That's fine. I just wanted to sanity check that it wasn't owned by OE, and that we would have to push some sshpass artifacts to it.